apps/api/src/app/portfolio/portfolio-calculator.ts

```diff
@@ -417,11 +422,38 @@ export class PortfolioCalculator {
       }
     }
 
-    const timelinePeriods: TimelinePeriod[][] = await Promise.all(
+    const timelineInfoInterfaces: TimelineInfoInterface[] = await Promise.all(
       timelinePeriodPromises
     );
+    const minNetPerformance = timelineInfoInterfaces
+      .map((timelineInfo) => timelineInfo.minNetPerformance)
+      .reduce((minPerformance, current) => {
+        if (minPerformance.lt(current)) {
```

Comment:
```
Error: [big.js] Invalid number
    at parse (/dev/ghostfolio/source/node_modules/big.js/big.js:142:13)
    at new Big (/dev/ghostfolio/source/node_modules/big.js/big.js:109:9)
    at Big.cmp (/dev/ghostfolio/source/node_modules/big.js/big.js:309:17)
    at Big.lt (/dev/ghostfolio/source/node_modules/big.js/big.js:479:17)
    at /dev/ghostfolio/source/dist/apps/api/webpack:/apps/api/src/app/portfolio/portfolio-calculator.ts:431:28
    at Array.reduce (<anonymous>)
    at PortfolioCalculator.<anonymous> (/dev/ghostfolio/source/dist/apps/api/webpack:/apps/api/src/app/portfolio/portfolio-calculator.ts:430:8)
    at Generator.next (<anonymous>)
    at fulfilled (/dev/ghostfolio/source/node_modules/tslib/tslib.js:111:62)
```

Reply:
This is problem is probably related:

```
TypeError: Cannot read property 'lt' of null
    at /dev/ghostfolio/source/dist/apps/api/webpack:/apps/api/src/app/portfolio/portfolio-calculator.ts:431:28
    at Array.reduce (<anonymous>)
    at PortfolioCalculator.<anonymous> (/dev/ghostfolio/source/dist/apps/api/webpack:/apps/api/src/app/portfolio/portfolio-calculator.ts:430:8)
    at Generator.next (<anonymous>)
    at fulfilled (/dev/ghostfolio/source/node_modules/tslib/tslib.js:111:62)
```

It happens shortly after `Missing value for symbol XYZ at 2021-04-14`.